main.cc

```diff
@@ -1142,7 +1142,9 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             });
 
             //This starts the update loop - but no real update happens until the data accessor is not initialized.
-            sl_controller.local().update_from_distributed_data(std::chrono::seconds(10));
+            sl_controller.local().update_from_distributed_data([cfg] () {
+                return cfg->service_levels_interval_ms();
+            });
```

Comment:
Nit: this is fine, but it would be better if this function returned a `std::chrono::duration` instead of `uint32_t` milliseconds.

Ideally only the config layer should care/know about the time unit. It needs that, because our only way to specify some duration in the config is by typing an integer in some agreed unit.

But other layers should communicate with `duration`s, because this removes the risk of confusing the unit somewhere down the line. (And it frees you from the need to document the unit).